main.js

```diff
@@ -0,0 +1,57 @@
+let bookCollection = [];
+
+const displayBooks = () => {
+  const bookList = document.getElementById('book-list');
+  bookList.innerHTML = '';
+
+  bookCollection.forEach((book) => {
+    const bookContainer = document.createElement('article');
+    const bookTitle = document.createElement('p');
+    const bookAuthor = document.createElement('p');
+    const removeBtn = document.createElement('button');
+    const hr = document.createElement('hr');
+
+    bookList.appendChild(bookContainer);
+    bookContainer.append(bookTitle, bookAuthor, removeBtn, hr);
+    bookTitle.textContent = book.title;
+    bookAuthor.textContent = book.author;
+    removeBtn.innerText = 'Remove';
+
+    const removeBook = (title) => {
+      bookCollection = bookCollection.filter((book) => book.title !== title);
+      localStorage.setItem('bookCollection', JSON.stringify(bookCollection));
+
+      displayBooks();
+    };
+
+    removeBtn.addEventListener('click', () => removeBook(book.title));
+  });
```

Comment:
- [ ] Nice job you did here. instead of removing by title, consider doing so by the `id` of each entry.
- try this:

```
const removeBook = (id) => {
  bookCollection = bookCollection.filter((book) => book.id !== id);
  localStorage.setItem('bookCollection', JSON.stringify(bookCollection));
  displayBooks();
};

removeBtn.addEventListener('click', () => removeBook(book.id));
```